### DISCUSSION TITLE
Please help me to collect anime characters ^^

### DISCUSSION THREAD

**Discussion by murakami10** [NAIST-SE/AttractingContributors#34] [General]
# No Anime, No Life.

Please tell me your favorite anime character.

https://github.com/murakami10/AttractingContributors-AnimeCharacter

**Comment by vittunyuta-aom** [comment]
added!! please help me https://github.com/aommoaGitHub/SuperFavoriteSongs

**Comment by tattiyaS** [comment]
hope you like it!

**Comment by noppadol-assava** [comment]
I commit pull request on your repo.
Please help me adding Food into the list.
https://github.com/NoppadolAssava/AttractingContributors---Food

**Comment by sergiodl-naist** [comment]
Added one of my all time favorites!
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers
